go-controller/pkg/ovn/ovn_test.go

```diff
@@ -98,12 +98,14 @@ func (o *FakeOVN) restart() {
 }
 
 func (o *FakeOVN) shutdown() {
-	close(o.stopChan)
 	o.watcher.Shutdown()
+	o.controller.nbClient.Close()
```

Comment:
No, `stopChan` will close the `Server` not the `Client`. `Close` method was recently addded to the libovsdb client. Since the user has a reference to the client I think it will be better to leave that `Close` under user control instead of doing it a bit under their feet through `stopChan`.